src/actinia_statistic_plugin/vector_sampling.py

```diff
@@ -0,0 +1,233 @@
+# -*- coding: utf-8 -*-
+"""
+Perform vector map sampling on a vector map layer based on input points.
+"""
+
+import pickle
+import tempfile
+import json
+from flask import jsonify, make_response
+from copy import deepcopy
+from flask_restful_swagger_2 import swagger
+from flask_restful_swagger_2 import Schema
+from actinia_core.models.response_models import ProcessingErrorResponseModel
+from actinia_core.rest.ephemeral_processing import EphemeralProcessing
+from actinia_core.rest.resource_base import ResourceBase
+from actinia_core.core.common.redis_interface import enqueue_job
+from flask.json import dumps
+from actinia_core.core.common.app import auth
+from actinia_core.core.common.api_logger import log_api_call
+from .response_models import VectorSamplingResponseModel
+
+
+__license__ = "GPLv3"
+__author__ = "Sören Gebbert, Markus Neteler"
```

Comment:
```suggestion
__author__ = "Markus Neteler"
```

This is a new file, do we need to add Sören here as author?

Reply:
fixed